ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,898 @@
+            *
+            * @type      {boolean}
+            * @since     next
+            */
+            enabled: true,
+
+            /**
+            * Fires when clicking on the breadcrumbs button.
+            * Two arguments are passed to the function. First breadcrumb button
+            * as an SVG element. Second is the breadcrumbs class,
+            * containing reference to the chart, series etc.
+            *
+            * ```js
+            * click: function(button, breadcrumbs) {
+            *   console.log(button);
+            * }
+            * ```
+            *
+            * Return false to stop default button's click action. // TO DO
+            *
+            * @type      {Highcharts.DrilldownBreadcrumbsClickCallbackFunction}
+            * @since     next
+            */
+
+            /**
+            * When the breadcrumbs is floating, the plot area will not move
+            * to make space for it. By default, the chart will not make space
+            * for the buttons.
+            * This property won't work when positioned in the middle.
+            *
+            * @type      {boolean}
+            * @since     next
+            */
+            floating: true,
+
+            /**
+            * A format string for the breadcrumbs button.
+            *
+            * @type      {string}
+            * @since     next
+            */
+            format: '{value}',
+
+            /**
+             * Callback function to format the breadcrumb text from scratch.
+             *
+             * @param {Highcharts.Breadcrumbs} breadcrumbs
+             *        The breadcrumbs instance
+             *
+             * @return {string}
+             *         Formatted text or false
+             * @since    next
+             */
+
+            /**
+            * Positioning for the button row. The breadcrumbs buttons
+            * will be aligned properly for the default chart layout
+            * (title,  subtitle, legend, range selector) for the custom chart
+            * layout set the position properties.
+            */
+            position: {
+                /**
+                 * Horizontal alignment of the breadcrumbs buttons.
+                 *
+                 * @type {Highcharts.AlignValue}
+                 */
+                align: 'left',
+
+                /**
+                 * Vertical alignment of the breadcrumbs buttons.
+                 *
+                 * @type {Highcharts.VerticalAlignValue}
+                 */
+                verticalAlign: 'top',
+
+                /**
+                 * The X offset of the breadcrumbs button group.
+                 */
+                x: 0,
+
+                /**
+                 * The Y offset of the breadcrumbs button group.
+                 */
+                y: 0
+            },
+
+            /**
+            * A CSS styles for all breadcrumbs.
+            *
+            * @type      {string}
```

Comment:
I have a few questions here :)
- Breadcrumbs' buttons or texts? Does it apply to separators too? Can I set for example `max-width` for the button, and text will get ellipsis?
- `@type` seems wrong ;) 
- add `@defaults`, maybe I'm wrong, but thatthis is necessary for [`CSSObject`](https://api.highcharts.com/class-reference/Highcharts.CSSObject) interface